features/YellowBox.feature

```diff
@@ -4,7 +4,73 @@ Feature: Managing Features with the YellowBox
   as a customer
   I want to be able approve and decline features
 
+  Background:
+    Given the store operates on a single channel in "EUR" currency
+    And I am on the homepage
+    And there is no yellowbox cookie
+
   @ui
-  Scenario: Visit the shop-page
-    When I am on the yellow-box page
-    Then I should see the yellow-box overlay
\ No newline at end of file
+  Scenario: I Visit the shop-page
+    When I am on the homepage
+    Then I should see the yellow-box overlay
+
+  @ui @javascript
+  Scenario: I open and close the overlay
+    Given I see the yellow-box overlay
+    When I click on the yellow-box symbol
+    Then I should see the expanded yellow-box overlay
+    And I click on the close button
+    Then I should not see the expanded yellow-box overlay
```

Comment:
The usage of two then-statements is in my opinion a smell for too much action for one Scenario. Maybe you should consider making two scenarios out of this.

One for opening the overlay and one for closing the overlay.